gnovm/Makefile

```
@@ -38,7 +38,7 @@ GOTEST_FLAGS ?= -v -p 1 -timeout=30m
 
 .PHONY: _test.cmd
 _test.cmd:
-	go test ./cmd/... $(GOTEST_FLAGS)
+	go test ./cmd/gno/... $(GOTEST_FLAGS)
```

```suggestion
	go test ./cmd/... $(GOTEST_FLAGS)
```

I suggest keeping `_test.cmd` the all-in-one cmd/ tester command. If needed, we can add subcommands.